packages/instrument-utils/src/storedInstrument/ManageAchInstrumentsTable/ManageAchInstrumentsTable.test.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { create } from 'react-test-renderer';
+import { fireEvent, render, screen } from '@testing-library/react';
+
+import {
+    createLocaleContext,
+    LocaleContext,
+    LocaleContextType,
+} from '@bigcommerce/checkout/locale';
+import { getInstruments, getStoreConfig } from '@bigcommerce/checkout/test-utils';
+
+import { isAchInstrument } from '../../guards';
+
+import ManageAchInstrumentsTable, {
+    ManageAchInstrumentsTableProps,
+} from './ManageAchInstrumentsTable';
+
+describe('ManageAchInstrumentsTable', () => {
+    let defaultProps: ManageAchInstrumentsTableProps;
+    let localeContext: LocaleContextType;
+
+    beforeEach(() => {
+        defaultProps = {
+            instruments: getInstruments().filter(isAchInstrument),
+            isDeletingInstrument: false,
+            onDeleteInstrument: jest.fn(),
+        };
+
+        localeContext = createLocaleContext(getStoreConfig());
+    });
+
+    it('matches snapshot with rendered output', () => {
+        const component = create(
+            <LocaleContext.Provider value={localeContext}>
+                <ManageAchInstrumentsTable {...defaultProps} />
+            </LocaleContext.Provider>,
+        );
+
+        expect(component).toMatchSnapshot();
+    });
+
+    it('renders instrument as row in table', () => {
+        render(
+            <LocaleContext.Provider value={localeContext}>
+                <ManageAchInstrumentsTable {...defaultProps} />
+            </LocaleContext.Provider>,
+        );
+
+        expect(screen.getByText('0000')).toBeInTheDocument();
+    });
+
+    it('triggers callback when delete button is clicked', () => {
+        render(
+            <LocaleContext.Provider value={localeContext}>
+                <ManageAchInstrumentsTable {...defaultProps} />
+            </LocaleContext.Provider>,
+        );
+
+        fireEvent.click(screen.getByText('Delete'));
```

Comment:
```suggestion
        userEvent.click(screen.getByText('Delete'));
```
Can we use `userEvent` instead?